### DISCUSSION TITLE
Question about the DISPLAY variable

### DISCUSSION THREAD

**Discussion by christianfosli** [WhitewaterFoundry/Fedora-Remix-for-WSL#164] [General]
I noticed on a fresh FedoraRemix36 installation on Windows 10 the DISPLAY variable got set to my local Windows IP address.
I do not have an X server installed and it caused `xsel` to hang which can have some unintended side effects
(see https://github.com/helix-editor/helix/discussions/3083).

So I'm just wondering if its intended to work like this. I guess I can just unset it in bash_profile so no big deal :smile:

**Comment by crramirez** [comment]
Hello @christianfosli,

Yes, it is set automatically to ease the integration with the XServer. The best approach is what you suggest to unset it in your bash_profile. Just keep it in mind when you upgrade to Windows 11.

Regards,
Carlos

**Reply by christianfosli** [reply]
Makes sense, thanks!
